recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -1,11 +1,19 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+# N.B. as Xen doesn't support partial .cfg as kernel does
+# we need to patch it to select GSX IMG as default
+SRC_URI_append = " \
+    file://0001-Make-GSX-IMG-coproc-default.patch \
+"
+
 ################################################################################
 # Renesas R-Car
 ################################################################################
 XEN_REL_rcar = "4.9"
 
 SRCREV_rcar = "${AUTOREV}"
 
-SRC_URI_rcar = "git://github.com/xen-troops/xen.git;protocol=https;branch=vgpu-dev"
+SRC_URI_rcar += "git://github.com/xen-troops/xen.git;protocol=https;branch=vgpu-dev"
```

Comment:
Why +=?

Reply:
because we have other SRC_URI manipilations in meta-xt-image's bbappend

Reply:
done